dts/bindings/mipi-dbi/zephyr,mipi-dbi-spi.yaml

```
@@ -27,6 +27,22 @@ properties:
     description: |
       Reset GPIO pin. Set high to reset the display
 
+  xfr-min-bits:
+    type: int
+    default: 8
+    description:
+      On rare types of SPI interfaces, discrete shift registers can be found
+      whose task is to convert the serial SPI bit stream to the parallel MCU
+      interface with clock and bit accuracy. Typically, these are 16 bits wide.
+
+      Use the macros not the actual enum value, here is the concordance list
```

```suggestion
      Use the macros, not the actual enum value. Here is the concordance list
```

Just a phrasing suggestion I received on the `mipi-mode` DT property, which has similar documentation wording

---

Ah, I see. Done this and also fix the same issue in "mipi-dbi-spi-device.yaml" – see commit 120d144e326bb06f3adc589dd0d43de936ddaafe